src/parse_file.py

```diff
@@ -156,6 +156,13 @@ def parse_file(f: typing.TextIO, options: Options) -> MIPSFile:
                         mips_file.new_jumptable_label(function_name)
                     else:
                         mips_file.new_function(function_name)
+                elif line.startswith('func'):
+                    # Other kind of function label.
+                    function_name: str = line[:-1]
+                    if re.match('L(_U_)?[0-9A-F]{8}', function_name):
```

Comment:
Hmm, IMO too much hardcoding here. Maybe we should just take `L_` as a required suffix and leave it at that?

Reply:
I just copied this from above, I'm not sure what this is checking for.

Reply:
I agree it's a bit much hard-coding, I'd be in favor of changing both occurrences to checking just that it starts with `L`. (Eventually we'll get rid of that magic as well, but that's a harder problem. `L` *is* well-established as meaning a local label, at least.)

Reply:
On second thought, this particular check is pretty silly. It checks whether the line stars with "L", after we have already established that it starts with "func".

Reply:
You're right, should I just assume that the statement is always false and remove the check?

Reply:
like what I now have?

Reply:
like what I now have?

Reply:
yep, looks good! maybe add `and line.endswith(":")` to the if condition?

also, you have a `'` while our coding style says `"` -- running `black` will fix that.